### DISCUSSION TITLE
[UI] selection protection disaffection

### DISCUSSION THREAD

**Discussion by afbagwell** [vladmandic/sdnext#1615] [General]
I noticed that recently something changed where now it is no longer possible to select the generated text of images on the right-hand side of the  From Text, From Image, and Process Image tabs. The only thing I can do is click the buttons to move said text between input boxes.

This makes me sad because sometimes I want to extract a piece of a prompt to use it in another one, and now there doesn't appear to be a way to do that in the UI. Unless there's some other way to access that text for copy-paste purposes.

**Comment by vladmandic** [comment]
I'm not aware of such thing

**Comment by afbagwell** [comment]
Yas... everything is OK now. Weird because the problem persisted through several browser restarts. Then rolled back to an earlier checkin, tried it, worked fine. Updated to latest, tried it, worked fine.

**Comment by afbagwell** [comment]
... and the problem has come back. Not sure what's going on. This time it's happening in Chrome instead of Firefox, because I switched over to that browser while investigating the full-screen mode problem.

**Reply by Tillerz** [reply]
Maybe it's a theme you are using for the UI, that somehow covers that area with an invisible layer or somethjing by accident? The <Ctrl-Shift-c> inspector in the browser might help here then. press that and then target that area with the mouse and see if the text area is clickable or if you get a layer above it or something. Or if you use a random theme, maybe try a basic one.

**Reply by afbagwell** [reply]
Good idea. I've been using the 'bethecloud/storj_theme' since changing themes became available. The problem didn't show up until recently.

But I took your advice and switched back to the default 'black-orange' theme.

The problem is still cropping up.

**Comment by vladmandic** [comment]
i just tried and cannot reproduce. one thing that comes to mind that it could be a color issue - text is getting selected, you just don't see selection? try selecting, -> ctrl+c to copy -> paste somewhere

**Reply by afbagwell** [reply]
Good suggestion. Tried it and not a color issue; copy-paste doesn't work. Still a good clue.

I haven't been able to pin down what the trigger is yet. When the browser starts there's no problem and sometimes it's fine through several image runs. I'll keep poking on it.

**Comment by i-liek-katz** [comment]
I've had this occur too. It happens every session I run, but I have extended sessions without doing a hard reboot. Might be related to the new image viewer, I think I noticed it happen around the time it was implemented. You also can not select text on metadata info of extra networks while the issue is present. In very long sessions, I even start seeing heavy delay between key strokes and the text printing into the field I'm typing. Nothing really pinpointed, but I can confirm I also have experienced this. I can try to think of ways to force reproduce it.

**Reply by rkiga** [reply]
Does it happen in a web browser that has no Browser Extensions installed? If you don't normally use Microsoft Edge, try it there. Or restart your browser in the mode that disables extensions. 
I couldn't edit text in the UI when using a spell check + grammar extension.

**Reply by i-liek-katz** [reply]
Yea, submitted the issue already. Used --safe in one test too. Issue currently occurs after you drag a zoomed in picture on the new image viewer.
